tools/clang_format_wrapper.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+"""Wrapper script for `git-clang-format` to avoid race conditions when
+merging/rebasing CPP files with merge conflicts."""
+from collections import namedtuple
+from os import environ as env
+from subprocess import check_output, CalledProcessError
+from typing import List
+import argparse
+import logging
+
+logging.basicConfig(format="[%(levelname)s] %(message)s", level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+
+class GitStatus:
```

Comment:
This shouldn't be class, a function suffices. Just return the list of added/modified paths from the property at the end.

Btw, I can recommend this talk:
https://www.youtube.com/watch?v=o9pEzgHorH0